arrange_schedule.py

```diff
@@ -769,9 +769,9 @@ def delete_old_cwb_img(db,server_dir,user_id):
     return error_list_id
 
 def mark_old_cwb_img(db,error_list_id):
-    for num2 in range(len(error_list_id)):
-        sql = "UPDATE image_data SET img_is_expire=1 WHERE img_is_expire=0 and img_is_delete=0 and img_id='" + str(error_list_id[num2]) + "'" 
-        db.cmd(sql)
+    for list_id in range(len(error_list_id)):
+        with ImageDao() as imageDao:
+                imageDao.markExpired(imgID=str(error_list_id[list_id]),markOldData=True)
 
```

Comment:
please help us modify the code as follow.
line772 `for list_id in error_list_id:`
line774 `imageDao.markExpired(imgID=list_id,markOldData=True)`